sdk/ml/azure-ai-ml/azure/ai/ml/entities/_job/sweep/sweep_job.py

```diff
@@ -148,7 +158,57 @@ def __init__(
         early_termination: Optional[Union[BanditPolicy, MedianStoppingPolicy, TruncationSelectionPolicy]] = None,
         queue_settings: Optional[QueueSettings] = None,
         **kwargs: Any,
-    ):
+    ) -> None:
+        """Sweep job for hyperparameter tuning.
+
+        :param name: Name of the job.
+        :type name: str
+        :param display_name: Display name of the job.
+        :type display_name: str
+        :param description: Description of the job.
+        :type description: str
+        :param tags: Tag dictionary. Tags can be added, removed, and updated.
+        :type tags: dict[str, str]
+        :param properties: The asset property dictionary.
+        :type properties: dict[str, str]
+        :param experiment_name:  Name of the experiment the job will be created under, if None is provided,
```

Comment:
Wording nit: sentence break:
```suggestion
        :param experiment_name:  Name of the experiment the job will be created under. If None is provided,
```